docs/src/snforge-advanced-features/fork-testing.md

```diff
@@ -21,37 +23,44 @@ We first need to define the contract's interface along with all the structures u
 
 There are two ways of configuring a fork:
 - by specifying `url` and block-related parameters in the `#[fork(...)]` attribute
-- or by passing a fork name defined in your `Scarb.toml` to the `#[fork(...)]` attribute
+- or by providing a fork name defined in your `Scarb.toml` to the `#[fork(...)]` attribute
+
+> 📝 **Note**
+> Using fork tests means `snforge` will make (often multiple) requests to the configured RPC URL.
+> These requests are relatively slow in comparison to standard tests.
+>
+> `snforge` can cache these requests automatically in `.snfoundry_cache` but only if `block_hash` or `block_number` is provided.
+> **Using `block_tag`, especially `"latest"` disables the caching functionality.**
 
 ### Configure a Fork in the Attribute
 
 It is possible to pass `url` and only one of `block_number`, `block_hash`, `block_tag` arguments to the `fork` attribute:
-- `url` (string literal) - RPC URL
-- `block_number` (hexadecimal number) - number of block which fork will be pinned to
-- `block_hash` (hexadecimal number) - hash of block which fork will be pinned to
-- `block_tag` (identifier) - tag of block which fork will be pinned to. Currently only `latest` is supported
+- `url` (string literal) — RPC URL
+- `block_number` (hexadecimal number) — number of a block which fork will be pinned to
```

Comment:
`block_number` doesn't need to be hexadecimal, what's more - usually nlock numbers are noted as decimal numbers
<img width="432" alt="image" src="https://github.com/user-attachments/assets/06980d22-cf3e-44e9-b305-149094e16c97" />

Reply:
Removed these entirely and added a note below